docs/UserGuide.adoc

```diff
@@ -438,12 +439,15 @@ __Constraints:__
 // end::find[]
 ==== Deleting tasks : `delete`
 
+Due to our fast changing lives, if your task was cancelled suddenly and you would like to delete the task from your task manager, simply use this delete function.
 To delete the specified task from the task manager, use: +
```

Comment:
Do note that the current version of the UG only uses a lead in line followed by the format. There's no statement with "use:". If you think it's necessary and would be too unclear otherwise, let me know, Otherwise, would appreciate if you could scrap this line.

Reply:
Ok I will change the sentence structuring to remove the word use